src/main/java/bio/terra/cli/command/group/ListUsers.java

```diff
@@ -31,8 +36,49 @@ protected void execute() {
 
   /** Print this command's output in text format. */
   private static void printText(List<UFGroupMember> returnValue) {
-    for (UFGroupMember groupMember : returnValue) {
-      groupMember.print();
+    TablePrinter<UFGroupMember> printer = ListUsers.UFGroupMemberColumns::values;
+    OUT.println(printer.print(returnValue));
+  }
+
+  /** Column information for fields in `resource list` output */
+  private enum UFGroupMemberColumns implements ColumnDefinition<UFGroupMember> {
+    EMAIL("EMAIL", g -> g.email, 45, LEFT),
```

Comment:
nit: you have room to make email much wider to allow for really long ones. You could go up to 80 safely I think.